### PR TITLE
docs(deployment): adjust with new version egg-scripts

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

https://github.com/eggjs/egg-scripts/pull/16

### CODE REVIEW

File: docs/source/zh-cn/core/deployment.md [hunk 1]
```diff
@@ -91,11 +91,14 @@ exports.cluster = {
...
 
 ```bash
-$ egg-scripts stop
+$ egg-scripts stop [--title=egg-server]
 ```
 
 该命令将杀死 master 进程，并通知 worker 和 agent 优雅退出。
 
+支持以下参数：
+- `--title=egg-server` 用于杀死指定的 egg 应用，未传递则会 killapp。
```
Comment (dead-horse) [inline docs/source/zh-cn/core/deployment.md]: > 未传递则会 killapp

这句有点看不懂

Comment (atian25) [reply docs/source/zh-cn/core/deployment.md]: 目前最新的 `egg-scripts stop` 会杀死机器上的所有 egg 应用

Comment (atian25) [reply docs/source/zh-cn/core/deployment.md]: killall 我写错了

Comment (atian25) [reply docs/source/zh-cn/core/deployment.md]: 改了

Comment (dead-horse) [reply docs/source/zh-cn/core/deployment.md]: 没 push？

Comment (atian25) [reply docs/source/zh-cn/core/deployment.md]: 额，好像在 GitHub 这边 update 后，push 就要 -f 才行。
推了

**Pull request merged**